Jenkinsfile_nightly

```diff
@@ -59,27 +59,20 @@ withNightlyPipeline("nodejs", product, component) {
     env.TEST_E2E_WAIT_FOR_ANGULAR = 'false'
     env.TEST_E2E_FAIL_FAST = 'false'
     env.TEST_E2E_ANNOTATION = params.NIGHTLY_TAG
-    enableFullFunctionalTest(120)
+    //enableFullFunctionalTest(120)
```

Comment:
Revert these comments before merging

Reply:
> Revert these comments before merging

Done. This has been taken care of, as part of https://github.com/hmcts/ia-ccd-e2e-tests/pull/409/files since this branch has been merged)